### PR TITLE
Project Duplicate Bug Fixing

### PR DESCRIPTION
**Pull request by nadiln**
Project Duplicate Problem fixing

## Description

When duplicating project, the duplicate is happening but the project tasks related to that particular project is not copying . instead of that some other tasks are copying to duplicated project. I found that duplicateid is parsed to Save.php but Save.ph is looking for relate_id which is a bug.

## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x ] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x ] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/Project/Save.php [hunk 1]
```diff
@@ -133,4 +133,4 @@
 	$_REQUEST['return_url'] = "index.php?module=Project&action=view_GanttChart&project_id=" . $return_id;
     handleRedirect($return_id,'Project');
 }
-?>
\ No newline at end of file
+?>
```
Comment (daniel-samson) [inline modules/Project/Save.php]: PSR-2: Please remove the ?>

**Comment by daniel-samson** [comment]
Also please sign the  Contributor License Agreement.

**Comment by nadiln** [comment]
Please check now... How to sign the Contributor License Agreement.?

**Comment by daniel-samson** [comment]
Select the details next to the clahub.

Here is the url:
http://www.clahub.com/agreements/salesagility/SuiteCRM

**Comment by nadiln** [comment]
@daniel-samson Thanks Daniel. I have signed that

**Comment by daniel-samson** [comment]
If anyone in the community would also test these changes then that would be helpful.

**Comment by nadiln** [comment]
@daniel-samson  Hi, have you tested this?

**Comment by daniel-samson** [comment]
@nadiln I haven't test. I have simply done code review. I have asked @Dillon-Brown to test it.

**Comment by Dillon-Brown** [comment]
@nadiln Works as expected.

**Comment by nadiln** [comment]
@Dillon-Brown @daniel-samson Thank you!.  @Dillon-Brown May I know how you are testing the changes? So in future i would be able to help someone by testing

**Comment by daniel-samson** [comment]
@nadiln When testing we simply try to install an instance or SuiteCRM and test the issue raised.

I personally look at the changes and search the code base for the areas it effects, then test those parts of the system.

**Comment by samus-aran** [comment]
This fix may possible resolve this issue.

https://github.com/salesagility/SuiteCRM/issues/2992

**Pull request merged**